### DISCUSSION TITLE
Unable to set device to Disabled in UI

### DISCUSSION THREAD

**Discussion by clowgg** [Koenkk/zigbee2mqtt#23644] [General]
When I check the 'Disabled' check box under 'Settings' of a device, the check mark immediately unchecks. Same goes for 'Retain'.
So I can't save it as disabled.

Versions: 1.39.1 commit: unknown
Frontend version: 0.7.4

**Comment by TheQue42** [comment]
Same here...Have you written a ticket?

**Reply by TheQue42** [reply]
ahh...

https://github.com/Koenkk/zigbee2mqtt/issues/23926
